old/industry.csv

```diff
@@ -208,8 +205,6 @@ Stephen M. Blackburn,Australian National University,http://users.cecs.anu.edu.au
 Minsuk Kahng,Oregon State University,https://minsuk.com,vieI1GYAAAAJ,Google
 Ashwin Machanavajjhala,Duke University,https://users.cs.duke.edu/~ashwin,A0hv6xwAAAAJ,Tumult Labs
 M. V. N. Ashwin Kumar,Duke University,https://users.cs.duke.edu/~ashwin,A0hv6xwAAAAJ,Tumult Labs
-Joseph E. Gonzalez,Univ. of California - Berkeley,http://www.cs.berkeley.edu/~jegonzal,gM2WW9UAAAAJ,Aqueduct
```

Comment:
This entry has been removed but not restored to UC Berkeley - is this intentional?

Reply:
@Eric-Wallace bump

Reply:
Hi, i just fixed this. joseph gonzalez has been removed from industry and restored to UC berkeley.